charts/k8s-infra/templates/_config.tpl

```diff
@@ -353,6 +353,15 @@ processors:
 
 {{- define "opentelemetry-collector.applyResourceDetectionInternalConfig" -}}
 {{- $config := mustMergeOverwrite (include "opentelemetry-collector.resourceDetectionInternalConfig" .Values | fromYaml) .config }}
+{{- if $config.service.pipelines.logs }}
```

Comment:
We don't need a `resourceDetectionInternal` anymore. Let's remove it and keep resourceDetection only.

Reply:
updated